Library/ViewModels/SettingsAccountViewModel.swift

```diff
@@ -101,6 +101,12 @@ SettingsAccountViewModelOutputs, SettingsAccountViewModelType {
 
     self.viewDidAppearProperty.signal
       .observeValues { _ in AppEnvironment.current.koala.trackAccountView() }
+
+    // Koala
+    updateCurrency.signal
+      .observeValues {
+        AppEnvironment.current.koala.trackChangedCurrency($0)
```

Comment:
This is fine...I think my previous comment was that we could have had this on 1 line like so..

```
updateCurrency.signal
  .observeValues { AppEnvironment.current.koala.trackChangedCurrency($0) }
```

^ up to you

Reply:
Oh ok. I particularly doesn't have preference, but changed it to have it in 1 line. Less one line to read 😄